USAGE.md

```diff
@@ -426,6 +435,55 @@ Get-GitHubUser
 
 ----------
 
+### Teams
+
+#### Getting teams in an Organization
+
+```powershell
+Get-GitHubTeam -OrganizationName Microsoft
```

Comment:
Can you lowercase the 'microsoft' examples here?

Reply:
Done